src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => {};
+import './UserInfo.scss';
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
+  );
```

Comment:
```suggestion
type Props = {
  user: User;
};

export const UserInfo: React.FC<Props> = ({ user }) => {
  return (
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
    </a>
  );
```
so right now we can update it